components/brave_news/browser/feed_v2_builder.cc

```diff
@@ -310,11 +313,94 @@ std::vector<mojom::FeedItemV2Ptr> GenerateChannelBlock(
   }
 
   std::vector<mojom::FeedItemV2Ptr> result;
-  result.push_back(mojom::FeedItemV2::NewCluster(
-      mojom::Cluster::New("channel", channel, std::move(article_elements))));
+  result.push_back(mojom::FeedItemV2::NewCluster(mojom::Cluster::New(
+      mojom::ClusterType::CHANNEL, channel, std::move(article_elements))));
   return result;
 }
 
+// Generate a Topic Cluster block
+// https://docs.google.com/document/d/1bSVHunwmcHwyQTpa3ab4KRbGbgNQ3ym_GHvONnrBypg/edit#heading=h.4vwmn4vmf2tq
+std::vector<mojom::FeedItemV2Ptr> GenerateTopicBlock(
+    const Publishers& publishers,
+    base::span<Topic>& topics) {
```

Comment:
Would it be better to have these by-reference arguments passed as `base::span<const Topic>`?

Reply:
Good call!